.github/workflows/push-build.yaml

```diff
@@ -56,3 +60,8 @@ jobs:
           GITHUB_REF: $ {{ env.GITHUB_REF }}
         run: |
           tests/scripts/build-release.sh
+
+      - name: setup tmate session for debugging when event is PR
+        if: failure() && github.event_name == 'pull_request'
+        uses: mxschmitt/action-tmate@v3
+        timeout-minutes: 60
```

Comment:
I think it might be good to leave this for debugging, but we can remove it in a follow-up if we want. Sound good?

Reply:
It will always be inactive since PRs don't trigger it, right?